README.md

```diff
@@ -1,7 +1,6 @@
 # laminas-diactoros
 
-[![Build Status](https://travis-ci.com/laminas/laminas-diactoros.svg)](https://travis-ci.com/laminas/laminas-diactoros)
-[![Coverage Status](https://coveralls.io/repos/github/laminas/laminas-diactoros/badge.svg)](https://coveralls.io/github/laminas/laminas-diactoros)
+[![Build Status](https://github.com/laminas/laminas-diactoros/workflows/Continuous%20Integration/badge.svg)](https://github.com/laminas/laminas-diactoros/actions?query=workflow%3A"Continuous+Integration")
```

Comment:
Could you apply the changes as described here? I think thats more future proof than using the workflow name.
https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-workflow-file-name

Reply:
done